Makefile

```diff
@@ -1,4 +1,5 @@
 GOPATH ?= $(shell go env GOPATH)
+NPM_DIR = $(shell npm config get prefix)
```

Comment:
Since we are only using `=`, not `:=`, the variable won't get evaluated if it is not used. You can still safely use most of the Makefile targets even if you don't have `npm` installed.